### PR TITLE
Release 0.2.3

### LINKED ISSUE #83: Release 0.2.3 Checklist
# Release Checklist

- [x] Update RELEASE.md - It should be in based on milestone description.
- [x] Commit the changes: 
```bash
git add RELEASE.md
git commit -m "Changelog for upcoming release <version>"
```
- [x] Final check `README` and `CONTRIBUTING`.
- [x] Bump LICENSE year
- [x] Bump version

### Test
- [x] Run the tests
- [x] Run the formatting
- [x] CI Check
```
python -m pytest
make all
```

### Setup.py
- [x] Update version number in setup.py (or version.py) (can also be minor or major)
- [x] Check if we need to upgrade `REQUIRES_PYTHON` or `REQUIRED` dependencies.
- [x] Check if local installation can be done.
- [x] Update `download_url` in setup
```bash
conda create --name temp
conda activate temp
conda install python==<version>
which pip
pip install .
```

### Documentation and Website
- [x] Check all documentations are updated
- [x] Update version label on documentation
  - [x] docs/conf.py
  - [x] Python version
- [x] Update version label on website
  - [x] Python version

```diff
- BEYOND THIS, EACH STEP SHOULD BE DONE IN SERIAL (STEP_BY_STEPS). -
```

### Last Edit
- [x] Rebase the commit history (should be done by only one person) right before the PR. (Not required)

### Update
1. [ ] Merge the branch to `master`
2. [ ] Release on PyPI by uploading `python setup.py upload` (make sure twine is installed)
3. [ ] Release on github (v<version>) "include v in front"

### Final Check
- [ ] Test that it pip installs
- [ ] Check the PyPI listing page to make sure that the README, release notes, and roadmap display properly.
- [ ] Edit the release-page on GitHub. Paste the release notes with a title for the release.
- [ ] Check Codcov ID and make sure they are running.
- [ ] Make mirror branch in private repo.

### PR DESCRIPTION
**Pull request by armantekinalp**
resolve #83

### CODE REVIEW

**Review by skim0119 — APPROVED** [review]
LGTM

File: examples/ExperimentalCases/ParallelConnectionExample/parallel_connection_example.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
+__doc__ = """Parallel connection example"""
+
+import numpy as np
+import sys
+
+# FIXME without appending sys.path make it more generic
+sys.path.append("../../../../")
```
Comment (bhosale2) [inline examples/ExperimentalCases/ParallelConnectionExample/parallel_connection_example.py:7]: verify if this works, I had to modify in damping branch to make it work.

**Review by bhosale2 — APPROVED** [review]
LGTM. @armantekinalp or @skim0119 please resolve the minor thing I pointed out.

File: examples/ExperimentalCases/ParallelConnectionExample/parallel_connection_example.py (hunk 1 above)
Comment (armantekinalp) [reply examples/ExperimentalCases/ParallelConnectionExample/parallel_connection_example.py]: Fixed

**Pull request merged**